### DISCUSSION TITLE
Εξέταση Μαθήματος

### DISCUSSION THREAD

**Discussion by MariaXristinaPapatheodwrou** [courses-ionio/sw#990] [Announcements]
Καλησπέρα σας,
Ονομάζομαι Μαρια Χριστίνα Παπαθεοδώρου (Α.Μ.: Π2018027) και είμαι Στ' εξάμηνο. Εξαιτίας ενός θέματος που είχα με το email μου δεν μπόρεσα να συμπληρώσω την δήλωση στο exams. Παρόλα αυτά έχω κάνει την δήλωση στο gramweb στην αρχή του εξαμήνου. Η βαθμολογία απο τις εργασίες θα περαστεί κανονικά; Αν ναι θα σας είμαι ευγνώμον! Σας ευχαριστώ εκ των προτέρων!

**Comment by epidrome** [comment]
όποιος κάνει δεκτό αίτημα ενσωμάτωσης για την τελική αναφορά 
θα βαθμολογηθεί αρκεί τα στοιχεία του να υπάρχουν στο γκραμγουεμπ
